addons/vagrant/Vagrantfile

```diff
@@ -40,18 +46,40 @@ Vagrant.configure("2") do |config|
         # Allow inbound, but only to our expected subnet. Allow outbound, but
         # only from our expected subnet. Allow traffic between guests. Deny
         # all other inbound. Deny all other outbound.
-        srv.vm.network "private_network", ip: details['mgmt_ip'], netmask: details['mgmt_netmask'], libvirt__dhcp_enabled: false, libvirt__forward_mode: "route"
+        srv.vm.network "private_network",
+                       :libvirt__network_name => networks[0]['name'],
+                       :ip => details['mgmt_ip'],
+                       :netmask => details['mgmt_netmask'],
+                       :libvirt__dhcp_enabled => false,
+                       :libvirt__forward_mode => 'route'
 
         # registration
         # libvirt__forward_mode: "none" mean:
         # Allow traffic between guests. Deny inbound. Deny outbound.
-        srv.vm.network "private_network", ip: details['reg_ip'], netmask: details['reg_netmask'], libvirt__dhcp_enabled: false, libvirt__forward_mode: "none"
+        srv.vm.network "private_network",
+                       :libvirt__network_name => networks[1]['name'],
+                       :ip => networks[1]['subnet'],
+                       :libvirt__dhcp_enabled => false,
+                       :libvirt__forward_mode => 'none',
```

Comment:
Maybe nat if you want to test the passthrough

Reply:
Good catch, done.